docs/topics/installing-mta-disconnected-environment.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="installing-mta-disconnected-environment_{context}"]
+= Installing {ProductShortName} on a disconnected environment
+
+On a connected device, first download and save the {ProductShortName} binary. Then download and save the Podman images required, you require the {ProductShortName} Podman image and the required provider image, you require.
```

Comment:
```suggestion
On a connected device, first download and save the {ProductShortName} binary. Then download and save the requisite Podman images.
```

This reads very confusing to me with all the "require".  It's not immediately clear to me the intention, but perhaps this suggestion may be used as inspiration.

What is meant by " _which include {ProductShortName} Podman image and the required provider image, you require_ "?  ProductShortName expands to "MTA", should this be directing to include the MTA CLI image?  

Perhaps instead of " _the required provider image, you require_  " , something like "the desired provider images"?